docs/paas-plus/dev/onboard-to-spryker-code-upgrader/prepare-a-project-for-spryker-code-upgrader.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Prepare a project for Spryker Code Upgrader
+description: Get your project ready to start using Spryker Code Upgrader
+template: concept-topic-template
+---
+
+Before you can start managing the upgrades of your project with Spryker Code Upgrader, fulfill the following prerequisites.
+
+## Update all Spryker modules to version 2022.04 or higher
+
+The Upgrader provides automatic minor and patch updates for the current version of each module. Since most updates are released for the latest module versions, modules need to be of the latest major version to receive updates.
+
+## Make your code is compliant with Spryker Quality Gate
+
+Spryker Quality Gate contains code checkers that ensure code compliance with Spryker development and customization guidelines. When an application is compliant with the guidelines, it can take minor and patch updates without breaking functionality, even if it’s highly customized.
```

Comment:
```suggestion
Spryker Quality Gate contains code checkers that ensure code compliance with Spryker development and customization guidelines. When an application is compliant with the guidelines, it can take minor and patch updates without breaking functionality, even if it is highly customized.
```